src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -11,7 +11,7 @@ import { selectAssets } from 'state/slices/selectors'
 
 import { useSwapper } from '../useSwapper/useSwapper'
 
-const ETHEREUM_CAIP19 = 'eip155:1/slip44:60'
+const ETHEREUM_ASSET_ID = 'eip155:1/slip44:60'
```

Comment:
Ditto, use `ethAssetId` from `caip` utils, we don't need to define a const here anymore